CONTRIBUTING.md

```diff
@@ -109,7 +109,7 @@ In brief:
 1. The published copy of the lesson is in the `gh-pages` branch of the repository
   (so that GitHub will regenerate it automatically).
   Please create all branches from that,
-  and merge the [master repository][repo]'s `gh-pages` branch into your `gh-pages` branch
+  and merge the [upstream repository][repo]'s `gh-pages` branch into your `gh-pages` branch
   before starting work.
   Please do *not* work directly in your `gh-pages` branch,
   since that will make it difficult for you to work on other contributions.
```

Comment:
This instruction about creating branches from `gh-pages` don't seem to reflect the reality of how this repo is being used, so I followed the pattern it looked like others were following.

Is this a vestigial instruction? `gh-pages` seems to only be updated by GitHub Actions in the way I'm used to.

Reply:
thanks @mfisher87 ! I *think* the most functional change is to update gh-pages to main in all cases (even if gh-pages is technically the published branch). Pinging @tobyhodges to verify.

Reply:
Thanks @karenword :)